lib/normalize2.js

```diff
@@ -97,12 +104,21 @@ module.exports = function (decl) {
                 }
             }
         }
+
+        if (isNotActual(mod) && val) {
+            let item = {};
+            item.block = block;
+            elem && (item.elem = elem);
+            item.modName = null;
+            item.modVal = val;
+            add(item, tech);
+        }
     }
 
     return res;
 
     function add(entity, tech) {
-        const str = naming.stringify(entity) + tech;
+        const str = entity.block + '__' + entity.elem + '_' + entity.modName + '_' + entity.modVal;
```

Comment:
Then, maybe something like this?
`Object.keys(entity).reduce((prev, key) => { return prev + entity[key] }, '') + tech;`